spark/v3.3/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestRewriteManifestsProcedure.java

```diff
@@ -75,6 +96,79 @@ public void testRewriteLargeManifests() {
         "Must have 4 manifests", 4, table.currentSnapshot().allManifests(table.io()).size());
   }
 
+  @Test
+  public void testRewriteLargeManifestsWithDateOrTimestampPartitionedTables() {
+    String[] scenarios = new String[] {"true", "false"};
+    String[] partitionColTypes = new String[] {"DATE", "TIMESTAMP"};
```

Comment:
I don't think we really need to test the case where Java 8 date/time classes are disabled. That's the normal case that we know works, right? I guess it's good to have for completeness?

Reply:
> That's the normal case that we know works, right? I guess it's good to have for completeness?

+1, the normal case works, added the normal case for increasing the coverage and completeness as we didn't had UT's for date / timestamp partitioned tables before.

Reply:
removed the ut's for java8 date/ time disabled, and seperated out timestamp & date partitioned table UT into diff methods.